### ISSUE TITLE
Electric Wiring messed up on copied bases

### ISSUE THREAD

**Issue by JohnTodd1971** [goatfungus/NMSSaveEditor#505]
Hi!

I really love the Save Editor and appreciate all the hard work.

But since the latest update, there is a problem with bases that have been backed up and then restored:

In the restored base,  all the electric wiring is disconnected by virtue of all the wires being "shifted over" by about a foot.  Therefore all connections are broken and no lights/equipment are getting power.

Everything else seems to be OK.

Again, thanks for all the hard work!
-John

**Comment by GeckoGT-Aztlan** [comment]
I also love the Save Editor and I have the same issue, some old backups seem to be ok.

Thank you for keeping this alive
-Ed

**Comment by biffrapper** [comment]
Yeah I have bases that have hundreds of wires with complex light arrangements and sequencing through bytebeat so I can't wire every time.  I've got some creative doors and switch setups, too.
